### PR TITLE
Feature/39713 - Minicart

### PR DESCRIPTION
**Pull request by Aekal**
Internal: 39713

### CODE REVIEW

File: components/03-modules/header/header.hbs [hunk 1]
```diff
@@ -26,10 +26,10 @@
...
                 {{/each}}
                 <div class="header__minicart">
-                    <a class="header-button popup-trigger"
-                       href="{{ minicartTrigger.link }}"
-                       aria-label="{{ minicartTrigger.ariaLabel }}"
-                       data-popuptrigger="popup-1"
+                    <button
+                        class="header-button popup-trigger"
+                        aria-label="{{ minicartTrigger.ariaLabel }}"
+                        data-popuptrigger="popup-minicart"
```
Comment (szafran89) [inline components/03-modules/header/header.hbs]: missing button type

File: components/03-modules/minicart-product/_minicart-product.scss [hunk 2]
```diff
@@ -106,21 +157,20 @@ $minicart-product__action-icon-color--active : $color-primary !default;
...
+        &:hover {
+            background-color: $minicart-product__action-button-bg;
         }
+    }
 
+    &__button-icon {
+        height: 24px;
+        width: 24px;
+        padding: 2px;
+        fill: $minicart-product__action-icon-color;
```
Comment (szafran89) [inline components/03-modules/minicart-product/_minicart-product.scss]: move above properties to variables

**Comment by szafran89** [comment]
Additionally please add colon and bigger space before input 
image

**Comment by szafran89** [comment]
Wrong color of link `View and edit cart`

**Comment by szafran89** [comment]
image

**Comment by szafran89** [comment]
Can we add loader before minicart items are rendered? For that you can create loader element.

File: components/03-modules/header/header.hbs (hunk 1 above)
Comment (Aekal) [reply components/03-modules/header/header.hbs]: Added

File: components/03-modules/minicart-product/_minicart-product.scss (hunk 2 above)
Comment (Aekal) [reply components/03-modules/minicart-product/_minicart-product.scss]: Done

**Comment by Aekal** [comment]
@szafran89 In the design I see black color of link `View and edit cart`, should I change it to green?

File: components/03-modules/minicart-product/_minicart-product.scss [hunk 3]
```diff
@@ -40,45 +59,57 @@ $minicart-product__action-icon-color--active : $color-primary !default;
...
     }
 
     &__name {
         margin-bottom: $minicart-product__spacer;
         font-weight: $font-weight-normal;
+        line-height: $minicart-product__name-line-height;
+    }
+
+    &__link {
+        text-decoration: none;
```
Comment (szafran89) [inline components/03-modules/minicart-product/_minicart-product.scss]: move to variable

File: components/03-modules/minicart-product/_minicart-product.scss [hunk 4]
```diff
@@ -40,45 +59,57 @@ $minicart-product__action-icon-color--active : $color-primary !default;
...
+        display: flex;
+        align-items: center;
+        margin: 0;
+        padding: 0;
+        font-family: $minicart-product__attributes-button-font-family;
+        font-size: $minicart-product__attributes-button-font-size;
+
+        .dropdown-list__icon {
+            position: static;
+            margin-left: $spacer;
```
Comment (szafran89) [inline components/03-modules/minicart-product/_minicart-product.scss]: move to variable

File: components/03-modules/minicart-product/_minicart-product.scss [hunk 5]
```diff
@@ -40,45 +59,57 @@ $minicart-product__action-icon-color--active : $color-primary !default;
...
     }
 
     &__attributes {
         margin-bottom: $minicart-product__spacer;
+    }
+
+    &__attributes-button {
+        display: flex;
+        align-items: center;
+        margin: 0;
```
Comment (szafran89) [inline components/03-modules/minicart-product/_minicart-product.scss]: move to variable

File: components/03-modules/minicart-product/_minicart-product.scss [hunk 6]
```diff
@@ -40,45 +59,57 @@ $minicart-product__action-icon-color--active : $color-primary !default;
...
+        padding: 0;
```
Comment (szafran89) [inline components/03-modules/minicart-product/_minicart-product.scss]: move to variable

File: components/03-modules/minicart-product/_minicart-product.scss (hunk 3 above)
Comment (Aekal) [reply components/03-modules/minicart-product/_minicart-product.scss]: Done

File: components/03-modules/minicart-product/_minicart-product.scss (hunk 4 above)
Comment (Aekal) [reply components/03-modules/minicart-product/_minicart-product.scss]: Done

File: components/03-modules/minicart-product/_minicart-product.scss (hunk 5 above)
Comment (Aekal) [reply components/03-modules/minicart-product/_minicart-product.scss]: Done

File: components/03-modules/minicart-product/_minicart-product.scss (hunk 6 above)
Comment (Aekal) [reply components/03-modules/minicart-product/_minicart-product.scss]: Done

**Pull request merged**